xml/System.Windows/ThemeMode.xml

```diff
@@ -28,8 +28,13 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>
+      <para>ThemeMode structure describes the Fluent theme mode to be applied on Application or a Window.</para>
+    </summary>
+    <remarks>
+      <para>This is an experimental API and may be modified or removed in future releases.</para>
+      <para>Since this is an experimental API, rather than creating a new instance of ThemeMode, use the static properties Light, Dark, System and None.</para>
```

Comment:
```suggestion
      <para>Since this is an experimental API, rather than creating a new instance of ThemeMode, use the static properties Light, Dark, System, and None.</para>
```